WMIAdventure/frontend/src/js/components/global/atoms/CompactCardView/CompactCardView.js

```diff
@@ -17,12 +18,16 @@ class CompactCardView extends React.Component {
         setTranslateX -> handle movement animation
         cardLevel -> information about card level
         cardImage -> card icon
+        cardsOrder -> to show particular card in BattleView and hide the rest
+        setOpacity -> to handle opacity animation in BattleView
      */
 
     state = {
         cardName: ' ',
         cardLevel: 1,
-        cardImage: null
+        cardImage: null,
+        compactCardOpacity: '1',
+        cardsOrder: this.props.cardsOrder,

```

Comment:
tak jak gdzieś wcześniej pisałem, ta nazwa jest myląca, trzeba wchodzić do komponentu wyżej, żeby się ogarnąć co to jest tak naprawdę